xlab/data/importer/iex/importer.py

```diff
@@ -56,3 +55,31 @@ def _make_data_base(self, symbol: str, data_date: datetime.datetime,
         data_entry.timestamp.FromDatetime(data_date)
         data_entry.updated_at.FromDatetime(now)
         return data_entry
+
+    def _get_date_range(self, start_date: datetime.datetime, end_date: datetime.datetime):
+        '''
+        calculate date range by counting no. of days between start and end days
+        '''
```

Comment:
What's the return type? Maybe link to the IEX Cloud Docs where the possible data range values are listed?